### PR TITLE
added posix and qurt apps to enable remote execution of shell command…

### PR DESCRIPTION
**Pull request by ndepal**
…s on qurt side from posix shell

commands are sent via muOrb to qurt, where they are executed and printed (i.e. visibile on mini-dm)

### CODE REVIEW

**Comment by tumbili** [comment]
Tested, works well!
@julianoes For some reason I was not able to do calibration even with your PR merged. But with this I was able to call commander calibrate directly on the qurt side.

**Comment by tumbili** [comment]
Example:
```
qshell commander calibrate accel
```

This will start accelerometer calibration on the qurt side. You will see the output in the debug terminal.

**Comment by mcharleb** [comment]
The functionality is fine. I thought about doing this by adding a shell command API in the px4muorb.idl file and using FastRPC to send the message to the DSP vs muorb, but this is fine as well.

**Comment by tumbili** [comment]
Merged, thanks!

**Comment by julianoes** [comment]
@ndepal Thanks, this is awesome! See one comment inline.

File: src/drivers/qshell/qurt/qshell_start_qurt.cpp [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+			/* this is not an error */
+			return 0;
+		}
+
+		PX4_INFO("before starting the qshell_entry task");
+
+		daemon_task = px4_task_spawn_cmd("qshell",
+						 SCHED_DEFAULT,
+						 SCHED_PRIORITY_MAX - 5,
+						 8192,
```
Comment (julianoes) [inline src/drivers/qshell/qurt/qshell_start_qurt.cpp]: I'm wondering. Do you need that much stack size for qshell? Or what is your thought here?

Comment (mcharleb) [inline src/drivers/qshell/qurt/qshell_start_qurt.cpp]: It does seem unnecessarily big.

Comment (ndepal) [inline src/drivers/qshell/qurt/qshell_start_qurt.cpp]: Could you elaborate on what you mean? I honestly simply based this on the [muorb example](https://github.com/PX4/Firmware/blob/master/src/platforms/qurt/tests/muorb/muorb_test_start_qurt.cpp#L86).

Comment (julianoes) [inline src/drivers/qshell/qurt/qshell_start_qurt.cpp]: @ndepal Ah ok, no worries. So the reserved stack size for the spawned command is 8192 bytes, however 1000 or 2000 would probably be plenty. On NuttX you can check max stack size used using the `top` command. On QURT, I'm not sure how to tackle this. @mcharleb?

**Comment by LorenzMeier** [comment]
Already cherry-picked to master, closing.

**Pull request closed without merging**